src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
```

Comment:
The message 'Excavator started working' could be more informative. It should indicate what type of machine is working, in this case, an Excavator. Consider including details about the work it's performing.